cycledash/static/js/runs/components/RunsPage.js

```diff
@@ -179,14 +179,16 @@ var ProjectTable = React.createClass({
     var numBams = this.props.bams.length;
     var numRuns = this.props.runs.length;
     var table;
-    if (this.state.bamsTable) {
+    if (this.state.bamsTable && numBams > 0) {
       table = <BamsTable bams={this.props.bams}
                          selectedBamId={this.state.selectedBamId}
                          createClickBamHandler={this.createClickBamHandler} />;
-    } else {
+    } else if (!this.state.bamsTable && numRuns > 0) {
       table = <RunsTable runs={this.props.runs}
                          selectedRunId={this.state.selectedRunId}
                          createClickRunHandler={this.createClickRunHandler} />;
+    } else {
+      table = <div className="project-empty-state">No files uploaded</div>;
```

Comment:
I think it would be better to say either "No runs yet" or "No BAMs yet", depending on the table you're looking at. We're not really expecting runs or BAMs to be "uploaded files".